browser/model/virtualbox.js

```diff
@@ -64,9 +64,7 @@ class VirtualBoxInstall extends InstallableItem {
                 resolve(output);
             });
           } else {
-            return new Promise((resolve, reject) => {
```

Comment:
I believe this is not the one Nick introduced - that one is still there (and it is commented).
See https://github.com/jrichter1/developer-platform-install/blob/promise-fix/browser/model/virtualbox.js#L82